reporter/html.js

```diff
@@ -861,20 +881,35 @@ QUnit.testDone( function( details ) {
 			toggleClass( assertList, "qunit-collapsed" );
 		} );
 
-		testItem.className = bad ? "fail" : "pass";
+		testItem.className = testPassed ? "pass" : "fail";
+
+		if ( details.todo ) {
+			let todoLabel = document.createElement( "em" );
+			todoLabel.className = "qunit-todo-label";
+			todoLabel.innerHTML = "todo";
+			testItem.insertBefore( todoLabel, testTitle );
+		}
 
 		time = document.createElement( "span" );
 		time.className = "runtime";
 		time.innerHTML = details.runtime + " ms";
 		testItem.insertBefore( time, assertList );
 	}
 
+	if ( !testPassed ) {
+		stats.failedTests++;
+	} else if ( details.todo ) {
+		stats.todoTests++;
+	} else {
+		stats.passedTests++;
+	}
```

Comment:
This need to be moved up inside the `else` block so skipped tests don't increment two `stats` counters.